packages/media-utils/src/get-partial-audio-data.ts

```diff
@@ -1,26 +1,27 @@
-import {ALL_FORMATS, AudioBufferSink, Input, UrlSource} from 'mediabunny';
+import type {InputAudioTrack} from 'mediabunny';
+import {AudioBufferSink} from 'mediabunny';
 
 // Audio frames might have dependencies on previous and next frames so we need to decode a bit more and then discard it.
 // The worst case seems to be FLAC files with a 65'535 sample window, which would be 1486.0ms at 44.1Khz.
 // So let's set a threshold of 1.5 seconds.
 const EXTRA_THRESHOLD_IN_SECONDS = 1.5;
 
 export type GetPartialAudioDataProps = {
+	track: InputAudioTrack;
 	fromSeconds: number;
 	toSeconds: number;
 	channelIndex: number;
 	signal: AbortSignal;
-	src: string;
-	isMatroska: boolean;
+	isMatroska?: boolean;
```

Comment:
Why are we making mandatory props optional?
This increases the chance for us to make mistakes

Reply:
Good catch! Thanks.